timesketch/frontend/src/components/AppNavbarSecondary.vue

```diff
@@ -73,10 +75,15 @@ limitations under the License.
                   >
                 </router-link>
               </li>
-              <li v-if="hasAttributeOntology('intelligence')" v-bind:class="{'is-active': currentPage === 'intelligence'}">
+              <li
```

Comment:
A little confusing that the formatting of Ontology is part of this PR but let it be :D

Reply:
This is the auto formatter that did this :) I had to change a few things above and that triggered this.